go/roothash/api/commitment/pool_test.go

```diff
@@ -204,6 +205,7 @@ func TestPoolSingleCommitmentTEE(t *testing.T) {
 
 	rt := &registry.Runtime{
 		ID:          rtID,
+		Kind:        registry.KindInvalid,
```

Comment:
You fixed the one above, but all the others should also be changed to `KindCompute`.

Reply:
fixed